vra/data_source_region.go

```diff
@@ -19,46 +18,54 @@ func dataSourceRegion() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"cloud_account_id": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Computed: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The id of the cloud account this region belongs to.",
```

Comment:
Could you update descriptions in `website/docs/d/vra_region.html.markdown` as well?

Reply:
Done!